staging/src/k8s.io/apiserver/pkg/util/flowcontrol/reqmgmt.go

```diff
@@ -154,19 +151,17 @@ func NewRequestManagementSystem(
 // optional and, if supplied, is kept informed of whenever a goroutine
 // is started or stopped or begins or finishes waiting --- except that
 // the configuration controller is not fully plumbed yet.
-func NewRequestManagementSystemWithPreservation(
+func NewRequestManagerWithPreservation(
 	informerFactory kubeinformers.SharedInformerFactory,
 	flowcontrolClient rmclientv1alpha1.FlowcontrolV1alpha1Interface,
-	queueSetFactory fq.QueueSetFactory,
 	serverConcurrencyLimit int,
 	requestWaitLimit time.Duration,
-	waitGroup *sync.WaitGroup,
 	preservingFlowSchemas []*rmtypesv1alpha1.FlowSchema,
 	preservingPriorityLevels []*rmtypesv1alpha1.PriorityLevelConfiguration,
 ) Interface {
 	reqMgmt := &requestManagementSystem{
```

Comment:
As long as we are renaming, I would rename the private data type along with the function that instantiates it.